src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -109,42 +111,47 @@ public void setElbowJointPower(double power) {
 
     /**
      * Gets the angle of the shoulder joint
+     *
      * @return angle of the shoulder joint [rad]
      */
     public double getShoulderJointAngle() {
-        return unitModel.toUnits(shoulderMainMotor.getSelectedSensorPosition());
+        return unitModel.toUnits(shoulderMainMotor.getSelectedSensorPosition()) - ArmConstants.SHOULDER_ABSOLUTE_ENCODER_OFFSET;
     }
 
     /**
      * Sets the angle of the shoulder joint
+     *
      * @param angle desired angle [degrees]
      */
     public void setShoulderJointAngle(double angle) {
         shoulderMainMotor.set(TalonFXControlMode.MotionMagic, unitModel.toTicks(Math.toRadians(angle)),
                 DemandType.ArbitraryFeedForward, shoulderFeedforward);
-        shoulderSetpoint = angle;
+        loggerInputs.shoulderSetpoint = angle;
```

Comment:
This should be updated in the `updateInputs()` method.

Reply:
but it needs to be the parameter angle